website/addons/base/__init__.py

```diff
@@ -88,46 +89,35 @@ def __init__(self, short_name, full_name, owners, categories,
         self.high_max_file_size = high_max_file_size
         self.accept_extensions = accept_extensions
 
+        NODE_SETTINGS_TEMPLATE_DEFAULT, USER_SETTINGS_TEMPLATE_DEFAULT = get_default_templates(settings.TEMPLATES_PATH)
```

Comment:
Why does `get_default_templates` need to be a function? It is only ever called with `settings.TEMPLATES_PATH`, so why not make these variables constants?

```
# top of file
NODE_SETTINGS_TEMPLATE_DEFAULT = os.path.join(settings.TEMPLATES_PATH, ...)
```